manage.py

```diff
@@ -251,14 +268,19 @@ def add(ctx, collections, note, name, tables_only, field_counts_option, field_li
     NOTE is your name and a description of your purpose
     """
     logger = logging.getLogger('ocdskingfisher.summarize.add')
-    logger.info('Arguments: collections=%s note=%s name=%s tables_only=%s',
-                collections, note, name, tables_only)
+    logger.info('Arguments: collections=%s note=%s name=%s tables_only=%s filter_tuple=%s',
+                collections, note, name, tables_only, filter_tuple)
 
     if not name:
         if len(collections) > 5:
             raise click.UsageError('--name is required for more than 5 collections')
         name = f"collection_{'_'.join(str(_id) for _id in sorted(collections))}"
 
+    if filter_tuple:
```

Comment:
I'm not sure whether this logic belongs here or in `sql_files` (in which case we would just pass the `filter_tuple` / `where` argument around).

Reply:
If we're adding an extra option in future for SQL/JSON Path Language, then I think we want to keep this where it is, so we can convert both options to a where fragment to pass around.